backend/migrations/20201003123223_students.js

```diff
@@ -0,0 +1,25 @@
+const { tableName } = require('../src/models/student')
+
+exports.up = (knex) => {
+  return knex.schema.createTable(tableName, (table) => {
+    table.increments('id').primary() // ToDo StudentID vs ID (postgres issue). Figure out uuid
```

Comment:
This might shed some light on using a custom id column: https://vincit.github.io/objection.js/api/model/static-properties.html#static-idcolumn

Reply:
Thank you!!!